### PR TITLE
Http acceptance tests

### PR DESCRIPTION
**Pull request by jmcarp**
So that the tests fail when they should 😓. Currently, test apps run with a process health check, which means cf only verifies that they start successfully and doesn't notice if they crash later. This patch refactors tests to run tiny web apps that we query via http, so we can verify that the apps can actually talk to k8s services.

To verify: introduce a bug into one of the test apps, push the app, and make sure a request to `/` fails.

### CODE REVIEW

File: infrastructure-aws-development.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - name: kubernetes-minion
+    properties:
+      aws:
+        cluster-tag: kubernetes-development
+- name: minion
+  jobs:
+  - name: kubernetes-minion
+    properties:
+      aws:
+        cluster-tag: kubernetes-development
```
Comment (cnelson) [inline infrastructure-aws-development.yml]: Did fixing the cluster tag fix the volume mounting problem?  If not what was it -- I want to understand that failure mode so we are sure it won't happen in prod

Comment (jmcarp) [inline infrastructure-aws-development.yml]: After I fixed the cluster tag, I restarted (twice) and recreated (once) the nodes and didn't get stuck in `ContainerCreating` again. I'll add more services and recreate again for fun.

Comment (cnelson) [inline infrastructure-aws-development.yml]: I think the right test would be to redeploy a version of the k8s release that's 1.5 and then deploy the 1.6 release  to see if it was an issue with the upgrade.

Comment (jmcarp) [inline infrastructure-aws-development.yml]: Destroyed the 1.6 deployment, deployed 1.5, created some test service instances, and upgraded. All pods came back after <5m with disks attached.

**Pull request merged**